react-components/stories/SearchHooks.stories.tsx

```diff
@@ -75,7 +75,6 @@ const StoryContent = ({ resources }: { resources: AddResourceOptions[] }): React
   const { data: assetSearchData } = useSearchMappedEquipmentAssetMappings(
     mainSearchQuery,
     filteredResources,
-    100,
     sdk
```

Comment:
I suppose this won't work - we still need to supply the limit?

Reply:
Are you suggesting we should not expose limit?